packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/activeDirectories/delete/federation-delete.html

```diff
@@ -1,7 +1,7 @@
 <div
     data-wizard
     data-wizard-title=":: 'dedicatedCloud_USER_AD_delete_title' | translate"
-    data-wizard-on-cancel="$ctrl.goBack.bind($ctrl)"
+    data-wizard-on-cancel="$ctrl.onGoBackClick.bind($ctrl)"
```

Comment:
Same as https://github.com/ovh/manager/pull/8158/files#r993037035
```suggestion
    data-wizard-on-cancel="$ctrl.boundOnGoBackClick"
```

Reply:
It's applied.